docs/template.md

```diff
@@ -20,22 +20,37 @@ import generate from "@babel/generator";
 import * as t from "@babel/types";
 
 const buildRequire = template(`
-  var IMPORT_NAME = require(SOURCE);
+  var %%importName%% = require(%%source%%);
```

Comment:
We should probably note that using this syntax requires `syntacticPlaceholders`?

I think we should keep the original docs and add a "check out this new hotness via this option" section right after it?

Reply:
No, it doesn't. If you are using `%%foo%%`, that option is automatically enabled.

Reply:
Sounds good.

> You can use either new or old placeholders by default, you just can't mix them.

Seems necessary to mention though?